exercises/exercise02-Task_Dependencies.ipynb

```diff
@@ -329,7 +336,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.1"
+   "version": "3.6.7"
```

Comment:
shall we revert that?

Reply:
It seems my jupyter notebook automatically changed it when I saved the changes. I will revert manually